hack/update-schemas.sh

```diff
@@ -24,55 +24,42 @@ GOFLAGS=""
 CRD_PATH=$(dirname "${0}")/../config/300-crds
 API_PATH=$(dirname "${0}")/../pkg/apis
 
-TEMP_DIR_LOGS=$(mktemp -d)
-
-for FILENAME in `find $CRD_PATH -type f`;
-do
-  echo "Gererating CRD schema for $FILENAME"
-
-  # NOTE: APIs for the group tekton.dev are implemented under ./pkg/apis/pipeline,
-  # while the ResolutionRequest from group resolution.tekton.dev is implemented under ./pkg/apis/resolution
-
-  GROUP=$(grep -E '^  group:' $FILENAME)
-  GROUP=${GROUP#"  group: "}
-  if [ "$GROUP" = "tekton.dev" ]; then
-    API_SUBDIR='pipeline'
-  else
-    API_SUBDIR=${GROUP%".tekton.dev"}
-  fi
-
+mapfile -d '' FILES < <(find "$CRD_PATH" -type f -name '*.yaml' -print0)
```

Comment:
can we add a comment above describing what exactly is happening here?

Reply:
This is just a result of using the best practice on how to read command output into an array: https://www.shellcheck.net/wiki/SC2207

So we just collect all existing crd yaml files from `config/300-crds/` into an array. We could add a comment if you think that it helps to understand the code. What do you think?

Reply:
yes please that would be good, thank you